assets/js/quick-bulk-edit.js

```diff
@@ -0,0 +1,28 @@
+( function ($) {
+	$( function () {
+		const wpInlineEditor = inlineEditPost.edit;
+
+		inlineEditPost.edit = function (id) {
+
+			wpInlineEditor.apply(this, arguments);
+
+			var postId = 0;
+			if (typeof (id) === 'object') {
+				postId = parseInt(this.getId(id));
+			}
+
+			if (postId != 0) {
+				var editRow = $('#edit-' + postId);
+				var postRow = $('#post-' + postId);
+
+				var statusCode = $('.srm_redirect_rule_status_code', postRow).text();
+				var forceHttps = $('.srm_redirect_rule_force_https', postRow).text();
+
+				$('select[name="srm_redirect_rule_status_code"]', editRow).val(statusCode);
+				if ('✓' === forceHttps) {
```

Comment:
@dhanendran I think we should not use symbols for comparison.


@dkotter do you have any thoughts?

Reply:
Yeah, this seems prone to breaking. Is there anything more specific we could target on, maybe we look for an empty string instead?

Reply:
@10up/open-source-practice curious if anyone else has insight on how to handle here?

Reply:
@ravinderk @dkotter @jeffpaul I'm using `x` character for comparison now.